tests/nested_pandas/nestedframe/test_nestedframe.py

```diff
@@ -320,6 +320,43 @@ def test_recover_from_flat():
     assert nf2.equals(nf)
 
 
+def test_from_lists():
+    """Test NestedFrame.from_lists behavior"""
+    nf = NestedFrame(
+        {"c": [1, 2, 3], "d": [2, 4, 6], "e": [[1, 2, 3], [4, 5, 6], [7, 8, 9]]}, index=[0, 1, 2]
```

Comment:
It would be nice to have a test case where the lists vary in length and include an empty list.

I'm feel that pack_lists handles this, but nice to have as a test (or to document if we couldn't handle it for whatever reason.)

Reply:
Added coverage for this!